test/Microsoft.AspNetCore.SignalR.Client.Tests/ServerSentEventsParserTests.cs

```diff
@@ -106,37 +107,34 @@ public void ParseSSEMessageIncompleteParseResult(string encodedMessage)
         [InlineData(new[] { "data: Hello, World\r\n", ":comment\r\n", "\r\n" }, "Hello, World")]
         public async Task ParseMessageAcrossMultipleReadsSuccess(string[] messageParts, string expectedMessage)
         {
-            using (var pipeFactory = new PipeFactory())
-            {
-                var parser = new ServerSentEventsMessageParser();
-                var pipe = pipeFactory.Create();
+            var parser = new ServerSentEventsMessageParser();
+            var pipe = new Pipe(new PipeOptions(new MemoryPool()));
```

Comment:
no need dispose any of these? (same in other places)

Reply:
I guess for cleanness sake we should dispose the `MemoryPool`